openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC2/certificate_estimation/HVAC2_ESC_calculation.py

```diff
@@ -236,12 +237,28 @@ class HVAC2_ESC_calculation(Variable):
     def formula(buildings, period, parameters):
       HVAC2_electricity_savings = buildings('HVAC2_electricity_savings', period)
       electricity_certificate_conversion_factor = 1.06
+      HVAC2_TCSPF_or_AEER_exceeds_benchmark = buildings('HVAC2_TCSPF_or_AEER_exceeds_benchmark', period)
+    
+      heating_capacity = buildings('HVAC2_heating_capacity_input', period)
+      zero_heating_capacity = ( heating_capacity == 0)
 
       result = np.rint(HVAC2_electricity_savings * electricity_certificate_conversion_factor)
+      
+      result_meet_elig = np.select([
+                            np.logical_not(zero_heating_capacity) * HVAC2_TCSPF_or_AEER_exceeds_benchmark,
```

Comment:
i did a workaround

essentially only check np.select conditions if heating capacity is not 0, otherwise return the default value (which is result)